tools/deploy.sh

```diff
@@ -6,28 +6,26 @@ DEPLOY_DIR="trash"
 SERVICE_NAME=""
 
 # Find deploy target and service name
-case $1 in
+case $GITHUB_REF in
```

Comment:
the deploy script uses the `$GITHUB_REF` env var given to us by github instead of a cli argument now.